plugins/modules/win_partition.ps1

```diff
@@ -1,6 +1,7 @@
 #!powershell
 
 # Copyright: (c) 2018, Varun Chopra (@chopraaa) <v@chopraaa.com>
+# Mountpoint feature added by Eriol (@aelfwine88)
```

Comment:
This isn't needed, we use the git attribution history to determine this information.